src/site/content/en/lighthouse-performance/resource-summary/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: Keep request counts low and transfer sizes small
+description: |
+  An overview of how high resource counts and large transfer sizes affect load performance, 
+  and strategies for reducing request counts and transfer sizes.
+updated: 2019-08-27
+web_lighthouse:
+  - resource-summary
+---
+
+The **Keep request counts low and transfer sizes small** audit tells you how many network requests 
+were made while your page loaded, as well as the total amount of data transferred.
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="resource-summary.jpg" 
+       alt="Keep request counts low and transfer sizes small">
+  <figcaption class="w-figcaption">
+    <b>Keep request counts low and transfer sizes small</b>
+  </figcaption>
+</figure>
+
+{% Aside %}
```

Comment:
FWIW, I think this aside might be trying to squeeze too much info into a single paragraph. I might try replacing this with two paragraphs in the body copy: one about how the **Total** row is calculated, and one about the **Third-party** row.